lib/ios-deploy.js

```diff
@@ -23,6 +23,7 @@ const APP_INSTALL_STRATEGY = Object.freeze({
 class IOSDeploy {
   constructor(udid) {
     this.udid = udid;
+    this.devicectl = new Devicectl(this.udid, log);
```

Comment:
we should also pass `log` to the device constructor. The idea here is that we use driver logger instead of static ones. the driver logger already has necessary attributes to be able to distinguish log lines from different paralllel driver sessions, while static ones don't.